### PR TITLE
Header

### PR DESCRIPTION
**Pull request by shorticins**

### CODE REVIEW

File: includes/header.php [hunk 1]
```diff
@@ -4,6 +4,45 @@
...
+
+    </header>
 <!-- END HEADER -->
+
+<?php
+//git push origin header
+
+//go to fork > plus sign > comment > check mark > three dots > push
+?>
+
```
Comment (thompson-ele) [inline includes/header.php]: Please remove these comments. Anything that you don't want all other groups to see should be removed before submitting the PR :)

File: includes/footer.php [hunk 2]
```diff
@@ -1,3 +1,32 @@
...
+            <p style="background-color: #2E9E47" class="four columns">Treat Street</p><!--replace with image-->
+
+            <a href="/" class="one column"><i class="fab fa-facebook"></i></a>
+            <a href="/" class="one column"><i class="fab fa-twitter"></i></a>
+            <a href="/" class="one column"><i class="fab fa-instagram"></i></a>
+            <a href="/" class="one column"><i class="fas fa-map-marker-alt"></i></a>
+
+        </div>
+
+        <hr>
```
Comment (thompson-ele) [inline includes/footer.php]: What is the purpose of this `<hr>` ? If it's just to add a border then we probably want to remove it and replace with CSS later. Take a look at the documentation for `<hr>`: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr

They describe the purpose of a horizontal rule (`<hr>`) as: "a thematic break between paragraph-level elements: for example, a change of scene in a story, or a shift of topic within a section."

Let me know what you think 😄

File: includes/footer.php [hunk 3]
```diff
@@ -1,3 +1,32 @@
 <!-- START FOOTER -->
+    <footer>
+        <div class="row">
+            <a href="/" class="one column"><i class="far fa-envelope"></i></a>
+            <a href="/" class="one column"><i class="fas fa-mobile-alt"></i></a>
+            <a href="/" class="one column"><i class="fas fa-question-circle"></i></a>
+            <a href="/" class="one column"><i class="fas fa-user-circle"></i></a>
+
+            <p style="background-color: #2E9E47" class="four columns">Treat Street</p><!--replace with image-->
```
Comment (thompson-ele) [inline includes/footer.php:9]: For now the inline-css is ok but we'll need to update this later on

**Review by thompson-ele — CHANGES_REQUESTED** [review]
I've left a couple of comments. Overall the structure looks pretty good! How does it feel submitting your first pull request 😺

File: includes/header.php (hunk 1 above)
Comment (shorticins) [reply includes/header.php]: Done - Shanen

Comment (shorticins) [reply includes/header.php]: These comments have been removed

File: includes/footer.php (hunk 2 above)
Comment (shorticins) [reply includes/footer.php]: The hr tag has been removed. Even if it was meant to be a thematic break, I think that is something that should be determined later.

File: includes/footer.php (hunk 3 above)
Comment (shorticins) [reply includes/footer.php:9]: Okay. Got it!

File: includes/header.php [hunk 4]
```diff
@@ -4,6 +4,40 @@
     <meta charset="utf-8">
     <title>Treat Street</title>
     <link href="/styleguide/styles.css" rel="stylesheet">
```
Comment (thompson-ele) [inline includes/header.php:7]: One last thing. Could you guys add this code above the styles.css <link> please? :)

`<link href="https://fonts.googleapis.com/css?family=Josefin+Sans:300,700i&display=swap" rel="stylesheet">`

**Review by thompson-ele — CHANGES_REQUESTED** [review]
If you guys could add in the Google Fonts link that I left in the comments that should be the last change. Thanks!

File: includes/header.php (hunk 4 above)
Comment (shorticins) [reply includes/header.php:7]: Okay. Sorry for the delay. The link has been added.

Comment (thompson-ele) [reply includes/header.php:7]: Thanks!

**Review by thompson-ele — APPROVED** [review]

**Pull request merged**